gustaf/io/meshio.py

```diff
@@ -72,18 +72,65 @@ def load(fname):
     return mesh
 
 
-def export(mesh, fname):
-    """Currently not implemented function.
+def export(mesh, fname, submeshes=None):
```

Comment:
Meshio is a very versatile library. If we want to provide an interface we should allow for kwargs to be passed to the save routine. For example, stl can be exported in ascii or binary format, which requires different flags. VTU supports different format versions etc...
```suggestion
def export(mesh, fname, submeshes=None, **kwargs):
```
or 
```suggestion
def export(mesh, fname, submeshes=None, save_options=None):
```